scripts/setup/install_viewer.sh

```diff
@@ -6,11 +6,14 @@ set -eu
 
 # Install cbmc-viewer
 
-if [[ $# -ne 1 ]]; then
-  echo "$0: Error: Specify the version to install"
+# Source kani-dependencies to get the CBMC viewer version
```

Comment:
nit: I'd sate the variable name, i.e. "to get CBMC_VERSION"

Reply:
Done.